kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -117,15 +117,15 @@ oriented data synchronization.
       shake: false,
       firstAttempt: true,
       complete: false,
-      correct: true,
+      correct: 0,
       itemError: false,
     }),
     methods: {
       updateAttemptLogMasteryLog({
         correct,
         complete,
         firstAttempt = false,
-        hinted = false,
+        hinted,
         answerState,
         simpleAnswer,
       }) {
```

Comment:
It makes the function call nicer, it also means that when you have a very long list of parameters, it i less cumbersome to call because it is the name of the parameter, not the specific order that determines the output. I think overall it is a more robust way and descriptive way to call a function.